### PR TITLE
fix actionlib state handling for non initialized action clients

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 1]
```diff
@@ -1500,7 +1501,7 @@ def get_state(self):
 		if self.client_mode != "": # not processing an actionlib client
 			return self.client_state
 		elif self.client == None:
-			return None
+			return GoalStatus.LOST
```
Comment (fmessmer) [inline cob_script_server/src/simple_script_server/simple_script_server.py:1503]: in what context did this lead to wrong behavior/did you detect this/did you have to change this?

Comment (floweisshardt) [reply cob_script_server/src/simple_script_server/simple_script_server.py:1503]: in a test script we're calling `sss.get_state()` for a non initialized action client and check for a valid actionlib GoalStatus. So returning a valid actionlib status (`LOST`) is better then returning `None`.

**Comment by fmessmer** [comment]
@floweisshardt rebased and made some additional changes...please re-test

**Review by floweisshardt — COMMENTED** [review]
tested with emulator: OK

@fmessmer good to merge.

**Review by fmessmer — APPROVED** [review]

**Pull request merged**